<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup_form</class>
 <widget class="QDialog" name="signup_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>577</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/logo.qrc">
    <normaloff>:/&lt;noprefix&gt;/4chan.png</normaloff>:/&lt;noprefix&gt;/4chan.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label_signin">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:24pt</string>
   </property>
   <property name="text">
    <string>SIGN UP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_captcha">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter The Text</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="captcha_fill">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>410</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_role">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:8pt
</string>
   </property>
   <property name="text">
    <string>Select Role</string>
   </property>
  </widget>
  <widget class="QComboBox" name="role">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Admin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>User</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_con_pass">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:8pt
</string>
   </property>
   <property name="text">
    <string>Confirm Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::PasswordEchoOnEdit</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="conf_password">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::PasswordEchoOnEdit</enum>
   </property>
  </widget>
  <widget class="QGraphicsView" name="captcha">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>410</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ActionsContextMenu</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="signup_button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>510</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pass">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:8pt
</string>
   </property>
   <property name="text">
    <string>Enter a Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_username">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:8pt</string>
   </property>
   <property name="text">
    <string>Enter a Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:8pt</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_warning">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="buddy">
    <cstring></cstring>
   </property>
  </widget>
  <widget class="QLabel" name="usrname_warning">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="buddy">
    <cstring></cstring>
   </property>
  </widget>
  <widget class="QLabel" name="captcha_warning">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>450</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="buddy">
    <cstring></cstring>
   </property>
  </widget>
  <widget class="QLabel" name="confpass_warning">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>380</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="buddy">
    <cstring></cstring>
   </property>
  </widget>
  <widget class="QLabel" name="pass_warning">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="buddy">
    <cstring></cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="show_pass">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>290</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/logo.qrc">
     <normaloff>:/&lt;noprefix&gt;/view.png</normaloff>:/&lt;noprefix&gt;/view.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="show_conf_pass">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>350</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/logo.qrc">
     <normaloff>:/&lt;noprefix&gt;/view.png</normaloff>:/&lt;noprefix&gt;/view.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="backbutton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/logo.qrc">
     <normaloff>:/&lt;noprefix&gt;/back-button.png</normaloff>:/&lt;noprefix&gt;/back-button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources/logo.qrc"/>
 </resources>
 <connections/>
</ui>
